pkg/operator/controller/extension/reconciler.go

```diff
@@ -156,20 +161,14 @@ func (r *Reconciler) reconcile(ctx context.Context, log logr.Logger, virtualClus
 		}
 	}
 
-	log.Info("Reconciling extension virtual resources")
-	if err := r.reconcileExtensionVirtualClusterResources(reconcileCtx, log, virtualClusterClientSet.Client(), extension); err != nil {
-		conditions.installed = v1beta1helper.UpdatedConditionWithClock(r.Clock, conditions.installed, gardencorev1beta1.ConditionFalse, ConditionReconcileFailed, err.Error())
-		return errors.Join(err, r.updateExtensionStatus(ctx, log, extension, conditions))
-	}
-
-	log.Info("Reconciling admission virtual resources")
-	if err := r.reconcileAdmissionVirtualClusterResources(reconcileCtx, log, virtualClusterClientSet, extension); err != nil {
+	log.Info("Reconciling ControllerRegistration and ControllerDeployment")
```

Comment:
Isn't "Reconciling" misleading here?
```suggestion
	log.Info("Deploying ControllerRegistration and ControllerDeployment")
```

Reply:
Thanks. I moved the log entry as well, to be more precise about deploying and deleting.